### DISCUSSION TITLE
Docker issue

### DISCUSSION THREAD

**Discussion by tp8441793** [availproject/avail#295] [Q&A]
I got this error when run node... Please help

/entrypoint.sh: line 35:     7 Illegal instruction     ${da_bin} --base-path /da/state --keystore-path ${da_keystore} --offchain-worker=Always --enable-offchain-indexing=true --execution native-else-wasm --name=${DA_NAME} --chain=${DA_CHAIN} --port=${DA_P2P_PORT} "$@"

**Comment by markopoloparadox** [comment]
Hello,

Would you be able to provide more information on what exactly commands you have run and what operating system you are using?

**Reply by sadness4me** [reply]
Hello,

I am using Windows 10 64 bit

**Comment by l3erryy99** [comment]
Am use Windows 10 same.

**Comment by Leouarz** [comment]
Hey, as this discussion is stalled, it will be close.
If needed, docs and binaries were recently update, you can now follow this documentation : https://docs.availproject.org/docs/operate-a-node/run-a-full-node

Especially the `Run an Avail node/Using Docker` section.
